web/mass_class/rtmp_bridge.py

```
@@ -0,0 +1,178 @@
+
+from ..models import MassClassStream
+
+logger = logging.getLogger(__name__)
+
+
+class RTMPBridgeConsumer(AsyncConsumer):
+    """
+    Consumer that handles WebRTC to RTMP conversion and YouTube streaming
+    """
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)